### DISCUSSION TITLE
【助睡】入睡前看更佳

### DISCUSSION THREAD

**Discussion by 290678** [share121/inter-knot#1282] [其他]
https://www.bilibili.com/video/BV1aC41177q8/?spm_id_from=333.880.my_history.page.click&vd_source=26ff47e5fcf0f2200263563708615829

**Comment by NatsukoYamamura** [comment]
B站的链接只需要到BV号就可以了
类似这样[https://www.bilibili.com/video/BV1aC41177q8](https://www.bilibili.com/video/BV1aC41177q8)

**Comment by lllliufeng** [comment]
可以这样
[入睡前看更佳](https://www.bilibili.com/video/BV1aC41177q8)

\[入睡前看更佳\](https://www.bilibili.com/video/BV1aC41177q8)

**Reply by 290678** [reply]
阿里嘎多
